README.md

```diff
@@ -636,6 +654,27 @@ To check and install the Cyberith SDK, please follow the steps for Cyberith SDK
 ### Configuration
 To configure the Cyberith SDK please follow the steps for Cyberith SDK mentioned in [here](https://github.com/robotology/walking-teleoperation/blob/master/docs/Dependencies.md).
 
+## Xsens
+Support for this dependency is enabled by the `ROBOTOLOGY_ENABLE_HUMAN_DYNAMICS` CMake option.
```

Comment:
I would invert the causality, I would document that the option `ROBOTOLOGY_USES_XSENS_MVN_SDK` option is only enabled when the `ROBOTOLOGY_ENABLE_HUMAN_DYNAMICS` CMake option is set to ON.

Reply:
What I wrote and what you said to me is equal! If you prefer, I can rephrase it as you mentioned.

Reply:
Yes, indeed I see what you mean. As long as we have the `ROBOTOLOGY_USES_XSENS_MVN_SDK` I am ok with what you wrote.